website/versioned_docs/version-5.x/ecosystem-css.md

```diff
@@ -0,0 +1,336 @@
+
+  <div id="single-spa-application:@org-name/project-name">
+    <div class="settings"></div>
+  </div>
+*/
+#single-spa-application\:\@org-name\/project-name .settings {
+  color: blue;
+}
+```
+
+### UI Frameworks
+
+Many popular UI frameworks have scoping built-in, or large ecosystems of open source libraries that help with scoping:
+
+#### React
+
+React CSS is quite diverse, with hundreds of options. Here are a few popular options that each result in component-scoped CSS:
+
+- [CSS Modules](https://github.com/css-modules/css-modules)
+- [Styled Components](https://styled-components.com/)
+- [Emotion](https://emotion.sh/docs/introduction)
+
+Also, in the single-spa community the Kremling library was authored, which scopes CSS while also unmounting it from the DOM when the React component unmounts:
+
+- [Kremling](https://github.com/CanopyTax/kremling)
+
+#### Angular
+
+[Angular Component Styles](https://angular.io/guide/component-styles) are built into Angular and facilitate scoping CSS to a component (and therefore, to its containing microfrontend).
+
+#### Vue
+
+Vue [Single File Components (SFC)](https://vue-loader.vuejs.org/spec.html) have built-in support for [Scoped CSS](https://vue-loader.vuejs.org/guide/scoped-css.html).
+
+#### Svelte
+
+Svelte scopes CSS classes by default ([Docs](https://svelte.dev/tutorial/styling)).
+
+### PostCSS Prefix Selector
+
+[PostCSS](https://postcss.org/) is a build tool that processes your CSS. It's often used via [postcss-loader](https://www.npmjs.com/package/postcss-loader).
+
+A particular PostCSS plugin called [postcss-prefix-selector](https://github.com/RadValentin/postcss-prefix-selector) can be very helpful to scope CSS to a microfrontend.
+With single-spa, each application is wrapped in a `<div id="single-spa-application:@org-name/project-name"></div>`, which can be used as a prefix to all CSS classes and selectors.
+
+The example code above in the Scoped CSS section shows the mechanics of how selector prefixing can accomplish scoping, and postcss-prefix-selector can do this automatically to
+all of your css.
+
+### Shadow DOM
+
+[Shadow DOM](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_shadow_DOM) is a browser API for scoping CSS. It is designed to be used by [Web Components](https://developer.mozilla.org/en-US/docs/Web/Web_Components), and is mentioned here as another viable option for scoping CSS.
+
+## Lazy Loading
+
+"Loading" CSS refers to downloading the CSS by inserting a `<link rel="stylesheet" href="/my-file.css">` element into the DOM, or by downloading a javascript file that inserts a `<style></style>` element into the DOM.
+
+"Lazy Loading" refers to only inserting the `<link>` or `<style>` elements into the DOM once they are needed, instead of all at once. In single-spa, this is during the [`load`](/docs/building-applications#load) or [`mount`](/docs/building-applications#mount) lifecycle function.
```

Comment:
```suggestion
"Lazy Loading" refers to only inserting the `<link>` or `<style>` elements into the DOM once they are needed, instead of all at once. In single-spa, this is during the [`load`](/docs/building-applications#load) or [`mount`](/docs/building-applications#mount) lifecycle functions.
```